src/accelerate/commands/launch.py

```diff
@@ -320,6 +326,7 @@ def simple_launcher(args):
 
     current_env = os.environ.copy()
     current_env["USE_CPU"] = str(args.cpu or args.use_cpu)
+    current_env["USE_MPS_DEVICE"] = str(args.use_mps_device).lower()
```

Comment:
```suggestion
    current_env["USE_MPS_DEVICE"] = str(args.use_mps_device)
```
We can't rely on users always setting lowercased values, so we should handle any casing.

Reply:
Hello, `use_mps_device` is a boolean argument using argparse action, so I think this should be fine.

Reply:
Done.